packages/connectivity/src/scp-cf/destination/get-subscriber-token.ts

```diff
@@ -72,34 +74,40 @@ async function retrieveUserToken(
 
 async function retrieveServiceToken(
   options: DestinationOptions,
-  isXsuaaJwt: boolean
+  decodedUserJwt: JwtPayload | undefined
 ): Promise<JwtPair | undefined> {
-  const jwt = getJwtForServiceToken(options, isXsuaaJwt);
+  const jwt = getJwtForServiceToken(options?.iss, decodedUserJwt);
```

Comment:
Why have a nullish check here, if the type of options isn't `options: DestinationOptions | undefined`?